seacrowd/utils/schemas/video.py

```diff
@@ -0,0 +1,22 @@
+"""
+General Video-to-Text Schema, including:
+- Video captioning
+- Video to text retrieval
+"""
+import datasets
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "video_path": datasets.Value("string"),
```

Comment:
add some notes here according to the discussion regarding URL must be publicly accessible, downloadable to obtain its videos, and safe from malicious files